Sources/NIO/EventLoop.swift

```diff
@@ -555,7 +659,7 @@ internal final class SelectableEventLoop: EventLoop {
         return thread.isCurrent
     }
 
-    public func scheduleTask<T>(in: TimeAmount, _ task: @escaping () throws -> T) -> Scheduled<T> {
+    public func scheduleTask<T>(at: NIODeadline, _ task: @escaping () throws -> T) -> Scheduled<T> {
```

Comment:
I'm happy enough with `at:` but why not just `deadline: NIODeadline`? Just a thought. @Lukasa / @normanmaurer  any input?

Reply:
`deadline:` works for me.